workers-site/index.js

```diff
@@ -1,17 +1,16 @@
-import { getAssetFromKV } from "@cloudflare/kv-asset-handler";
+import { getAssetFromKV } from '@cloudflare/kv-asset-handler'
 
-addEventListener("fetch", event => {
-  event.respondWith(handleEvent(event));
-});
+addEventListener('fetch', event => {
+  event.respondWith(handleEvent(event))
+})
 
 async function handleEvent(event) {
   try {
-    return await getAssetFromKV(event);
+    return await getAssetFromKV(event)
   } catch (e) {
-    let pathname = new URL(event.request.url).pathname;
-    return new Response(`"${pathname}" not found`, {
+    let pathname = new URL(event.request.url).pathname
+    return new Response(e.message || e.toString(), {
```

Comment:
i'm curious what we are prescribing to users; this can be great for debugging, but i imagine is not the behavior a user would want for a production site. do we suggest handling 404 responses in some customized way?

Reply:
I'd say that's a nice to have. I actually had that in my original implementation and in my [8track PR](https://github.com/jrf0110/8track/blob/6386efbd82cae66d7544b5389428753996b0294d/src/middleware/kv-static.ts#L12), but this solution is less complicated by throwing an error (@EverlastingBugstopper's idea which I agreed with ). Then in the template, we direct users towards the proper way of handling that error. 

This goal of this approach is to give the _best debug experience_ the worst thing that would happen is clients see a legit/useful error message (`"blah/index.html  was not found` over` "blah/"  was not found`.

Reply:
let's land this. we can improve it later. i think it's a massive improvement over the current, which would also show to end users.